broker/src/main/java/org/apache/rocketmq/broker/schedule/ScheduleMessageService.java

```diff
@@ -161,15 +152,15 @@ public void start() {
                 }
             }
 
-            this.deliverExecutorService.scheduleAtFixedRate(() -> {
+            scheduledPersistService = new ScheduledThreadPoolExecutor(1,
```

Comment:
Why do we need to change the code here? In my opinion, removing the scheduleAtFixedRate in the constructor is enough.

Reply:
I have change the code. I think persist task should execute by scheduledPersistService but not deliverExecutorService.